crypto-ffi/bindings/jvm/src/main/kotlin/com/wire/crypto/CoreCrypto.kt

```diff
@@ -8,7 +8,7 @@ import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 
 /** Wrap a `CoreCrypto` instance in a `CoreCryptoClient` instance. Should largely be invisible to end-users. */
-fun CoreCrypto.lift() = CoreCryptoClient(this)
+fun CoreCryptoFfi.lift() = CoreCrypto(this)
 
 /** Opens an existing core crypto client or creates a new one if one doesn't exist at the `keystore` path */
 suspend operator fun CoreCrypto.Companion.invoke(
```

Comment:
Now that we're renaming the uniffi class to `CoreCryptoFfi`, we don't need this external declaration of an invoke method on the companion object anymore. We can just declare a secondary constructor on `CoreCrypto` which takes these methods and produces a `CoreCrypto`.